apache/trafficcontrol
# Dev CiaB: Prefix Traffic Monitor profile name with TM profile prefix

This PR fixes an issue in CDN in a Box for Developers where `monitoring.json` does not contain any parameters.

This makes the *TR Ultimate Test Harness* GitHub Actions workflow pass again.
<hr/>

## Which Traffic Control components are affected by this PR?

- CDN in a Box for Developers

## What is the best way to verify this PR?

1. Start CDN in a Box for Developers
   ```shell
   docker-compose up
   ```

2. Take a new snapshot of the *dev* CDN

3. Go to https://localhost:444/configs/monitoring and verify the config contains these parameters:
   * `health.polling.interval`
   * `heartbeat.polling.interval`
   * `peers.polling.interval`
   * `tm.polling.interval`

## If this is a bugfix, which Traffic Control versions contained the bug?

- master

## PR submission checklist
- [x] This PR fixes tests
